### DISCUSSION TITLE
Adding tmux destroys with the robby russell theme in oh my zsh! Why?

### DISCUSSION THREAD

**Discussion by nishu-murmu** [ohmyzsh/ohmyzsh#11584] [Help] [answered]
**![Screenshot_2023-03-26_23-36-22](https://user-images.githubusercontent.com/71123337/227795357-94a1e93c-f866-47c9-aec4-18519dfc3790.png)

![Screenshot_2023-03-26_23-32-54](https://user-images.githubusercontent.com/71123337/227795193-e403be82-6dba-47b0-a98c-46a073b75a96.png)

It is **_fish shell_** but same is the case with the **bash shell!**

**Comment by mcornella** [comment, marked as answer]
Oh My Zsh only works for zsh. If you switch to other shells such as `bash` or `fish` OMZ won't get loaded.

**Reply by mattdibi** [reply]
Hi, I have the same issue as OP but I'm on zsh

<img width="418" alt="immagine" src="https://github.com/ohmyzsh/ohmyzsh/assets/22748355/f90d0211-3e52-4ab3-9c1f-d115dedc9ed8">

expected output:

<img width="418" alt="immagine" src="https://github.com/ohmyzsh/ohmyzsh/assets/22748355/69f8a1cb-e6e5-4343-8059-9dbfb945204c">

**Reply by mattdibi** [reply]
For anybody with the same issue: starting tmux with `u` flag solves this issue for me.

```sh
tmux -u
```

> [-u](http://man.openbsd.org/OpenBSD-current/man1/tmux.1#u)      Write UTF-8 output to the terminal even if the first environment variable of LC_ALL, LC_CTYPE, or LANG that is set does not contain "UTF-8" or "UTF8"

Apparently `utf8` was causing issues, see:
- https://superuser.com/questions/651868/weird-behaviour-rendering-on-tmux-in-osx
- https://askubuntu.com/questions/410048/utf-8-character-not-showing-properly-in-tmux
